### PR TITLE
Dev2021

### PR DESCRIPTION
**Pull request by kemalkongar**
Fixes issue #662 by updating the document in `inputChangeOpacity ()`. Minor updates to .gitignore files.

### CODE REVIEW

**Review by raviraina — COMMENTED** [review]
LGTM

**Review by GabbyHalpin — COMMENTED** [review]
NOICEEEE

File: src/DisplayPanel/DisplayPanel.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ class DisplayPanel implements DisplayInterface {
    * Update SVG based on visualization settings
    */
   updateVisualization (): void {
-    DisplayControls.setOpacityFromSlider(this.className);
+    // DisplayControls.setOpacityFromSlider(this.className);
```
Comment (JRegimbal) [inline src/DisplayPanel/DisplayPanel.ts:85]: You can just delete lines you comment out - that's the beauty of version control!

File: src/DisplayPanel/DisplayControls.ts [hunk 2]
```diff
@@ -106,24 +106,27 @@ function setOpacityControls (meiClassName: string): void {
   function inputChangeOpacity (): void {
     opacityOutput.value = opacitySlider.value;
     lastGlyphOpacity = Number(opacitySlider.value);
-    (document.querySelector('.' + meiClassName) as HTMLElement)
-      .style.opacity = (Number(opacityOutput.value) / 100.0).toString();
+    try {
+      ((document.querySelectorAll('.' + meiClassName)) as NodeListOf<HTMLElement>).forEach(g => {
+        g.style.opacity = (Number(opacityOutput.value) / 100.0).toString();
+      });
+    } catch (e) {}
```
Comment (JRegimbal) [inline src/DisplayPanel/DisplayControls.ts:113]: If this shouldn't fail silently, add a `console.warn` so debugging isn't painful later if.

File: .gitignore [hunk 3]
```diff
@@ -6,7 +6,6 @@
 /deployment/public/img
 /deployment/public/assets
 /deployment/public/Neon
-/deployment/public/uploads/*
```
Comment (JRegimbal) [inline .gitignore:9]: Why the changes to the gitignores? This doesn't seem to be related to the linked issue.

**Review by JRegimbal — COMMENTED** [review]
Some minor comments/questions on the actual code changes made, but this does seem to strictly fix the issue. That being said, there does seem to be a related problem where newly loaded pages do not inherit the correct glyph opacity (see attached video). It should be easy to fix that by reapplying opacity after a new page is rendered.

https://user-images.githubusercontent.com/6963142/122224279-f3b8a700-ce81-11eb-9ff4-babd99871eba.mp4

File: src/DisplayPanel/DisplayControls.ts (hunk 2 above)
Comment (JRegimbal) [reply src/DisplayPanel/DisplayControls.ts:113]: s/if/on
Always have coffee before doing a code review.

File: src/DisplayPanel/DisplayPanel.ts (hunk 1 above)
Comment (raviraina) [reply src/DisplayPanel/DisplayPanel.ts:85]: Was my plan but was convinced otherwise, we'll take a look at the opacity inheritance issue and remove the commented code

File: .gitignore (hunk 3 above)
Comment (kemalkongar) [reply .gitignore:9]: It was for  development purposes, we got multiple pages running in the `yarn build` by importing `mei` and `manifests` folders from the `gh-pages` branch. It was easier to deal with the imports using local `.gitignore` files rather than ignoring the entire `uploads` directory.

**Comment by raviraina** [comment]
@JRegimbal made a couple changes, should fix your new issue
